test/worker_test.py

```diff
@@ -131,6 +132,14 @@ def run(self, result=None):
         if time.time != self.time:
             time.time = self.time
 
+    def setUp(self):
+        # ensure that other tests don't leak information into these via the task register
+        luigi.task_register.Register.clear_instance_cache()
+
+    def tearDown(self):
+        # ensure these tasks don't leak into others via the task register
+        luigi.task_register.Register.clear_instance_cache()
```

Comment:
Instead of setUp and tearDown, did you consider using the [LuigiTestCase](https://github.com/spotify/luigi/blob/6cf7091326af6eba64965767cef83f62a1bff53b/test/helpers.py#L141) helper?